config/config-example.js

```diff
@@ -136,6 +135,17 @@ Y929lRybWEiKUr+4Yw2O1W0CAwEAAQ==
 -----END PUBLIC KEY-----
 `;
 
+/**
+ * routes - where Pokemon Showdown is hosted.
+ *   Don't change this setting - there aren't any other options right now
+ */
+exports.routes = {
+	root: 'pokemonshowdown.com',
+	client: 'play.pokemonshowdown.com',
+	dex: 'dex.pokemonshowdown.com',
+	replays: 'replay.pokemonshowdown.com',
+};
```

Comment:
I'd suggest ``app: 'sim.smogon.com',`` for the simulator server.

Reply:
Also ``logs: 'logs2.psim.us:8080,'``.

Reply:
Neither of these are actually used anywhere in either this or the client repo? Also, actually hosting a logs server is also pretty esoteric currently (should https://github.com/Zarel/Pokemon-Showdown-Client/tree/log-viewer be merged into HEAD instead of being an orphan branch?). I can see adding those two for 'completeness' and so that if anyone *does* want to link to them from the repo in the future its more obvious that they should instead be using `${Config.routes.logs}` for example, but I'm also not sure its not 'YAGNI'?

Reply:
I would suggest staging server URLs here, instead of the official URLs.

Reply:
> I would suggest staging server URLs here, instead of the official URLs.

Can you elaborate here? Are you talking about just the proposed `logs` and `app` routes? Things will break currently if `Config.routes` doesn't point to eg. the correct official `client` or `root` URLs otherwise?

Reply:
I'm pretty sure logs only appear in private code (and the config file?) anyway.

Reply:
> Can you elaborate here?

Nevermind, I thought this was the client. These are good defaults (although you'll break every existing server on update, forcing them to manually update their config).

Reply:
> (although you'll break every existing server on update, forcing them to manually update their config)

Ugh, you're right. Is there a way around this?

Reply:
Set the default if it doesn't exist.

`if (!Config.routes) Config.routes = ...`

Reply:
Probably would want to check each route TBH, in case only some are specified to be as safe as possible? Is `server/index.js` the place for this?

Reply:
Possibly. I've wanted a config loader for a while (client has one now!) but haven't made one yet.

Reply:
``app`` would be used by #5515, and yeah ``logs`` would be for the private chat plugin.